package.json

```diff
@@ -30,7 +30,7 @@
     "start": "node app.js",
     "start-with-fixtures": "node utils/sails_with_fixtures.js",
     "debug": "node debug app.js",
-    "test": "./node_modules/.bin/istanbul cover node_modules/mocha/bin/_mocha",
+    "test": "./node_modules/.bin/istanbul cover node_modules/mocha/bin/_mocha && node_modules/.bin/eslint --quiet .",
```

Comment:
This should also work and is easier to read (IMO) `"test": "istanbul cover _mocha && eslint --quiet ."`